### DISCUSSION TITLE
Add multi-channel communication in your RAG app: Session Resources

### DISCUSSION THREAD

**Discussion by multispark** [microsoft/RAG_Hack#56] [Announcements]
This is the resources and discussion thread for this live stream: https://aka.ms/raghack/multi-channel
It will be updated with links after the session.

Recording: [Video](https://www.youtube.com/watch?v=xnmok9Al7iA)
Slides: [PPT](https://github.com/microsoft/RAG_Hack/blob/main/slides/Add%20multi-channel%20communication%20to%20your%20RAG%20app.pptx)
Code samples:
* https://aka.ms/whatsAppRag
* https://aka.ms/cakeShopSample

Please ask any follow-up questions here, and the speakers will see them.

**Comment by zzulueta** [comment]
In APAC, Whatsapp is not so popular. We rely more on Messenger and Viber. When do you think would Azure Communication Services support these channels? I recall the Azure Bot Service and Copilot Studio already supporting Messenger

**Comment by trumhemcut** [comment]
I can't access the powerpoint file with information rights protected. Could you please help?

**Reply by Yash-Kavaiya** [reply]
@trumhemcut 

You can find some sesson ppts in here :- https://speakerdeck.com/pamelafox

To view a `.pptx` file, you have several options:

1. **Local Repository**: 
   - Clone the repository to your local machine.
   - Open the `.pptx` file using PowerPoint or any other supported software.

2. **GitHub Codespaces**:
   - Open the repository in GitHub Codespaces.
   - Download and install a `.pptx` viewer extension within GitHub Codespaces.

3. **Office Web Access**:
   - If you do not have Microsoft Office installed, you can access Office applications online.
   - Upload the `.pptx` file to the web version of Office and view it there.

**Comment by Yash-Kavaiya** [comment]
**@multispark** Could you please provide any sample code for integrating with **Telegram** and **Discord**?
